l3kernel/l3luatex.dtx

```diff
@@ -208,9 +234,7 @@
 % \begin{macro}[EXP]{\lua_now:n, \lua_now:e}
 % \begin{macro}{\lua_shipout_e:n, \lua_shipout:n}
 % \begin{macro}[EXP]{\lua_escape:n, \lua_escape:e}
-%   Wrappers around the primitives. As with engines other than \LuaTeX{}
-%   these have to be macros, we give them the same status in all cases.
-%   When \LuaTeX{} is not in use, simply give an error message/
+%   Wrappers around the primitives.
```

Comment:
Any particular reason for this change?

Reply:
The non-LuaTeX error code needs to come after the definition of `\lua_load_module:n`, but it would be odd to define `\lua_load_module:n` before we wrap the primitives, so I broke the original `macrocode` block into two.

Reply:
Oh sorry, I missed that you moved to comment to later. I thought that it just got removed which confused me.